src/actions/getCurrentVersionsByPackage.js

```diff
@@ -0,0 +1,10 @@
+export function getCurrentVersionsByPackage({ packageJson, props }) {
```

Comment:
And if we do not return an array, we should name this `getCurrentPackageVersions` (without "ByPackage" which indicates a list in other actions)...